03-transactions.md

```diff
@@ -223,6 +271,45 @@ If a revoked commitment transaction is published, the remote node can spend this
 
 The sending node can use the HTLC-timeout transaction to timeout the HTLC once the HTLC is expired, as shown below. This is the only way that the local node can timeout the HTLC, and this branch requires `<remotehtlcsig>`, which ensures that the local node cannot prematurely timeout the HTLC since the HTLC-timeout transaction has `cltv_expiry` as its specified `locktime`. The local node must also wait `to_self_delay` before accessing these funds, allowing for the remote node to claim these funds if the transaction has been revoked.
 
+### Leasee Offered HTLC Outputs (`option_will_fund`)
```

Comment:
As discussed on the mailing list, I think we should remove this, because we have no way of actually exchanging signatures for those transactions unless we add a round-trip, which isn't desirable. The leasee should adapt their `max_accepted_htlc` and `max_htlc_value_in_flight_msat` if they're afraid that the seller will cheat.